modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -120,7 +120,7 @@ export default class TileHeader {
     header: {[key: string]: any},
     parentHeader: TileHeader,
     basePath: string,
-    extendedId: string = ''
+    extendedId = ''
```

Comment:
Why drop the type?

Reply:
![image](https://user-images.githubusercontent.com/18549629/121495835-9d131f00-c9e2-11eb-8ee0-3b1fd545c1ea.png)

Reply:
Good catch. I will try to disable that rule.